src/composables/useLoading.js

```diff
@@ -0,0 +1,12 @@
+export default function useLoading() {
+  const nuxtApp = useNuxtApp()
+  const isLoading = ref(true)
+
+  nuxtApp.hook('page:start', () => {
+    isLoading.value = true
+  })
+  nuxtApp.hook('page:finish', () => {
+    isLoading.value = false
+  })
+  return isLoading
```

Comment:
Please return the ref in an object to make it usable in the same way as all other composables (e.g. from vueUse)

Reply:
Ok, done